libraries/botbuilder-ai/botbuilder/ai/qna/utils/qna_card_builder.py

```diff
@@ -67,13 +67,6 @@ def get_qna_prompts_card(result: QueryResult, card_no_match_text: str) -> Activi
             for prompt in result.context.prompts
         ]
 
-        # Add No match text

```

Comment:
- `get_qna_prompts_card` gets called for multi-turn qna dialog scenarios
- It's not an active learning scenario
- remove active learning NO MATCH appending from `get_qna_prompts_card`
- the NO MATCH response is still present within `get_suggestions_card`, which is what is called for active learning prompt scenarios